### ISSUE TITLE
runner требует наличия add

### ISSUE THREAD

**Issue by asosnoviy** [silverbulleters/vanessa-runner#276]
runner help

{Модуль C:\Program Files (x86)\OneScript\lib\vanessa-runner\src\Классы\КомандаТестирование_xUnitFor1C.os / Ошибка в строке: 22 / Библиотека не найдена add}

**Comment by artbear** [comment]
Да, все верно, для выполнения команд vanessa и xunit требуется наличие add

**Comment by artbear** [comment]
Не совсем верно, конечно, что при выполнении команды "help" требуется наличие пакета add.

но это особенности реализации согласно шаблону oscript-app-template

**Comment by asosnoviy** [comment]
Вариант 1. (не очень) сунуть адд в жесткие зависимости
Вариант 2. Сделать, чтобы адд требовался только для комманд связанных с ним

**Comment by asosnoviy** [comment]
И это баг, а не вопрос...
